content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -63,30 +63,31 @@ In **extended annotated CSV**, measurements, fields, and values and their types
+|      | `--debug`           | Output errors to stderr                                                                      |             |                       |
+|      | `--encoding`        | Character encoding of input (default `UTF-8`)                                                | string      |                       |
+|      | `--error-file`      | Path to a file used for recording rejected row errors                                        | string      |                       |
+| `-f` | `--file`            | File to import                                                                               | stringArray |                       |
+|      | `--format`          | Input format (`lp` or `csv`, default `lp`)                                                   | string      |                       |
+|      | `--header`          | Prepend header line to CSV input data                                                        | string      |                       |
+| `-h` | `--help`            | Help for the `write` command                                                                 |             |                       |
+|      | `--host`            | HTTP address of InfluxDB (default `http://localhost:8086`)                                   | string      | `INFLUX_HOST`         |
+|      | `--max-line-length` | Maximum number of bytes that can be read for a single line (default `16000000`)              | integer     |                       |
+| `-o` | `--org`             | Organization name (mutually exclusive with `--org-id`)                                       | string      | `INFLUX_ORG`          |
+|      | `--org-id`          | Organization ID (mutually exclusive with `--org`)                                            | string      | `INFLUX_ORG_ID`       |
+| `-p` | `--precision`       | Precision of the timestamps (default `ns`)                                                   | string      | `INFLUX_PRECISION`    |
+|      | `--rate-limit`      | Throttle write rate (examples: `5 MB / 5 min` or `1MB/s`).                                   | string      |                       |
+|      | `--skip-verify`     | Skip TLS certificate verification                                                            |             |                       |
+|      | `--skipHeader`      | Skip first *n* rows of input data                                                            | integer     |                       |
+|      | `--skipRowOnError`  | Output CSV errors to stderr, but continue processing                                         |             |                       |
+| `-t` | `--token`           | Authentication token                                                                         | string      | `INFLUX_TOKEN`        |
+| `-u` | `--url`             | URL to import data from                                                                      | stringArray |                       |
 
 ## Examples
 
```

Comment:
We need to add examples for both line protocol and CSV. Add the following examples in their respective sections as well as links in the Examples TOC.

~~~md
##### Write line protocol from a compressed file
```sh
# The influx CLI assumes files with the .gz extension use gzip compression 
influx write \
  --bucket example-bucket \
  --file path/to/line-protocol.txt.gz

# Specify gzip compression for gzipped files without the .gz extension
influx write \
  --bucket example-bucket \
  --file path/to/line-protocol.txt.comp \
  --compression gzip
```

##### Write annotated CSV data from a compressed file
```sh
# The influx CLI assumes files with the .gz extension use gzip compression 
influx write \
  --bucket example-bucket \
  --file path/to/data.csv.gz

# Specify gzip compression for gzipped files without the .gz extension
influx write \
  --bucket example-bucket \
  --file path/to/data.csv.comp \
  --compression gzip
```
~~~